tracking-migration/src/main/java/io/customer/tracking/migration/MigrationAssistant.kt

```diff
@@ -36,6 +37,7 @@ class MigrationAssistant private constructor(
             // Re-identify old profile to new implementation
             logger.debug("Migrating existing profile...")
             migrateExistingProfile()
+            migrateExistingDevice()
```

Comment:
Because we use device token in identify method, shouldn't we be saving device token before identifying profile? 🤔